packages/langium/src/parser/async-parser.ts

```diff
@@ -16,8 +16,8 @@ import { Emitter } from '../utils/event.js';
 /**
  * Async parser that allows cancellation of the current parsing process.
  *
- * @remark The sync parser implementation is blocking the event loop, which can become quite problematic for large files.
- * @remark The default implementation is not actually async. It just wraps the sync parser in a promise. A real implementation would create worker threads or web workers to offload the parsing work.
```

Comment:
Was there a reason for removing the remark annotations?

Reply:
They are not recognized by JSDoc, so TypeDoc was complaining about them

Reply:
Interesting, hmm, could you tweak it a bit to be `remark: ` instead? It would be nice to retain the intent.

Reply:
Alright, fair enough

Reply:
It's possible to define your own tags, see https://typedoc.org/guides/tags/#defining-tags.
Here is [an example](https://typedoc.org/guides/tags/#block-tags) that introduce `@remarks` tag.

Reply:
Oh, I didn't notice that part of the docs. In that case, we can either define a custom tag or simply use `@remarks` which is already recognized by TypeDoc (though there can only be one per comment). I personally vote for the latter, what do you think?

Reply:
Actually, I just noticed now that `@remarks` is already used in `lsp/node-kind-provider.ts`, so I'm guessing `@remark` in async parser was just a typo?

Reply:
I would be on board with the latter, even if it's just a single `@remarks` 👍

Reply:
Done ✅